.gitsplit.yml

```diff
@@ -28,6 +28,8 @@ splits:
     target: "https://${GH_TOKEN}@github.com/opentelemetry-php/contrib-auto-wordpress.git"
   - prefix: "src/Context/Swoole"
     target: "https://${GH_TOKEN}@github.com/opentelemetry-php/context-swoole.git"
+  - prefix: "src/Instrumentation/bin"
```

Comment:
I think the directory should be `src/Instrumentation/Installer`

Reply:
ok, I was thinking about that, but was not sure if this will not be misleading (as someone can think of it as another library that we instrumenting).

Reply:
mmm, good point. What about moving it up to the top level, as `AutoInstrumentationInstaller` or something like that?

Reply:
Yes, that might be right place. I was also thinking about that, however could not choose meaningful name, `AutoInstrumentationInstaller` is ok for me.